src/core/ddsi/include/dds/ddsi/ddsi_cfgelems.h

```diff
@@ -0,0 +1,1994 @@
+      "transport_priority greater than or equal to this element's value will "
+      "be delivered synchronously from the \"recv\" thread, all others will "
+      "be delivered asynchronously through delivery queues. This reduces "
+      "latency at the expense of aggregate bandwidth.</p>")),
+  STRING("SynchronousDeliveryLatencyBound", NULL, 1, "inf",
+    MEMBER(synchronous_delivery_latency_bound),
+    FUNCTIONS(0, uf_duration_inf, 0, pf_duration),
+    DESCRIPTION(
+      "<p>This element controls whether samples sent by a writer with QoS "
+      "settings transport_priority >= SynchronousDeliveryPriorityThreshold "
+      "and a latency_budget at most this element's value will be delivered "
+      "synchronously from the \"recv\" thread, all others will be delivered "
+      "asynchronously through delivery queues. This reduces latency at the "
+      "expense of aggregate bandwidth.</p>"),
+    UNIT("duration_inf")),
+  INT("MaxParticipants", NULL, 1, "0",
+    MEMBER(max_participants),
+    FUNCTIONS(0, uf_natint, 0, pf_int),
+    DESCRIPTION(
+      "<p>This elements configures the maximum number of DCPS domain "
+      "participants this Cyclone DDS instance is willing to service. 0 is "
+      "unlimited.</p>")),
+  INT("AccelerateRexmitBlockSize", NULL, 1, "0",
+    MEMBER(accelerate_rexmit_block_size),
+    FUNCTIONS(0, uf_uint, 0, pf_uint),
+    DESCRIPTION(
+      "<p>Proxy readers that are assumed to sill be retrieving historical "
+      "data get this many samples retransmitted when they NACK something, "
+      "even if some of these samples have sequence numbers outside the set "
+      "covered by the NACK.</p>")),
+  ENUM("RetransmitMerging", NULL, 1, "never",
+    MEMBER(retransmit_merging),
+    FUNCTIONS(0, uf_retransmit_merging, 0, pf_retransmit_merging),
+    DESCRIPTION(
+      "<p>This elements controls the addressing and timing of retransmits. "
+      "Possible values are:</p>\n"
+      "<ul><li><i>never</i>: retransmit only to the NACK-ing reader;</li>\n"
+      "<li><i>adaptive</i>: attempt to combine retransmits needed for "
+      "reliability, but send historical (transient-local) data to the "
+      "requesting reader only;</li>\n"
+      "<li><i>always</i>: do not distinguish between different causes, "
+      "always try to merge.</li></ul>\n"
+      "<p>The default is <i>never</i>. See also "
+      "Internal/RetransmitMergingPeriod.</p>"),
+    VALUES("never","adaptive","always")),
+  STRING("RetransmitMergingPeriod", NULL, 1, "5 ms",
+    MEMBER(retransmit_merging_period),
+    FUNCTIONS(0, uf_duration_us_1s, 0, pf_duration),
+    DESCRIPTION(
+      "<p>This setting determines the size of the time window in which a "
+      "NACK of some sample is ignored because a retransmit of that sample "
+      "has been multicasted too recently. This setting has no effect on "
+      "unicasted retransmits.</p>\n"
+      "<p>See also Internal/RetransmitMerging.</p>"),
+    UNIT("duration"),
```

Comment:
This is also something that was not present in the q_config.c fields.?

Reply:
See other comment about units etc.